test/performance/broker-rmq/100-broker-perf-setup.yaml

```diff
@@ -110,10 +111,11 @@ spec:
 apiVersion: eventing.knative.dev/v1
 kind: Trigger
 metadata:
-  name: rabbitmq-broker-perf
+  name: rabbitmq-trigger-perf
```

Comment:
can't figure out myself why to rename, please tell me :-)

Reply:
Since the resource kind is a trigger it could be confusing to call it a broker (happened to me xD)